contributor_docs/es/steward_guidelines.md

```diff
@@ -0,0 +1,384 @@
+# Directrices para Administradores
+
+
+Ya sea que recién te hayas unido a nosotros como administrador, seas un mantenedor experimentado de p5.js, o estés en algún punto intermedio, esta guía contiene información, así como consejos y trucos que te ayudarán a contribuir de manera efectiva a p5.js. La mayor parte de lo que se escribe aquí son pautas, a menos que se indique lo contrario, lo que significa que puedes adaptar las prácticas mostradas aquí para que se ajusten a tu flujo de trabajo.
+
+## Tabla de Contenidos
+
+- [Issues](steward_guidelines.md#issues)
+  - [Bug report](steward_guidelines.md#bug-report)
+  - [Feature request](steward_guidelines.md#feature-request)
+  - [Feature enhancement](steward_guidelines.md#feature-enhancement)
+  - [Discussion](steward_guidelines.md#discussion)
+- [Pull Requests](steward_guidelines.md#pull-requests)
+  - [Simple fix](steward_guidelines.md#simple-fix)
+  - [Bug fix](steward_guidelines.md#bug-fix)
+  - [New feature/feature enhancement](steward_guidelines.md#new-feature-feature-enhancement)
+  - [Dependabot](steward_guidelines.md#dependabot)
+- [Build Process](steward_guidelines.md#build-process)
+  - [Main build task](steward_guidelines.md#main-build-task)
+  - [Miscellaneous tasks](steward_guidelines.md#miscellaneous-tasks)
+- [Release Process](steward_guidelines.md#release-process)
+- [Tips & Tricks](steward_guidelines.md#tips--tricks)
+  - [Reply templates](steward_guidelines.md#reply-templates)
+  - [GitHub CLI](steward_guidelines.md#github-cli)
+  - [Managing notifications](steward_guidelines.md#managing-notifications)
+
+---
+
+
+## Issues
+
+Alentamos a la mayoría de las contribuciones de código fuente a comenzar con un problema, y como tal, los issues son el lugar donde la mayoría de las discusiones tendrán lugar. Los pasos a seguir al revisar un problema dependerán del tipo de problema que sea. El repositorio utiliza [GitHub issue templates] (https://github.com/processing/p5.js/blob/main/.github/ISSUE_TEMPLATE) para organizar mejor los diferentes tipos de problemas y alentar a los autores de problemas a proporcionar toda la información relevante sobre sus problemas. El primer paso al revisar el issue a menudo será revisar la plantilla completada y determinar si necesita información adicional por ejemplo, porque algunos campos no se completaron o se utilizó la plantilla incorrecta.
```

Comment:
traducir el texto del link de Github por favor

Reply:
Siempre que se deje una palabra en inglés como _Issue_ por favor ponerlo en itálicas

Reply:
Entendido

Reply:
itálicas aplicadas.